src/calculateRentalCost.js

```diff
@@ -27,8 +27,15 @@
  * @return {number}
  */
 
-function calculateRentalCost(days) {
-  // write code here
+function calculateRentalCost(day) {
+  switch (true) {
+    case (day >= 3 && day < 7):
+      return 40 * day - 20;
+    case (day >= 7):
+      return 40 * day - 50;
+    default:
+      return 40 * day;
+  }
 }
```

Comment:
```suggestion
function calculateRentalCost(day) {
  switch (true) {
    case (day >= 3 && day < 7):
      return 40 * day - 20;
    case (day >= 7):
      return 40 * day - 50;
    default:
      return 40 * day;
  }
}
```
solution with 'switch statement' looks complicated, use 'ternary' operator or 'if/else' statement instead

Reply:
I would like to train my switch skill, so could this code be like this?